ang/civiawards/award-creation/directives/review-fields/review-fields-table.directive.js

```diff
@@ -175,7 +175,11 @@
      * Open the Popup to Select Review fields
      */
     function openReviewFieldSelectionPopup () {
-      dialogService.open('ReviewFields', '~/civiawards/award-creation/directives/review-field-selection.html', $scope, {
+      if (dialogService.dialogs.ReviewFields) {
```

Comment:
is this here to avoid opening the same popup twice? But, why would it be opened twice?

Reply:
Yes. If the user clicks the Add Review Panel, while its already open, it will create the popup multiple times.